js/views/modals/purchase/Moderators.js

```diff
@@ -98,7 +98,16 @@ export default class extends baseVw {
                     // don't add profiles that are not moderators. The ID list may have peerIDs
                     // that are out of date, and are no longer moderators.
                     if (eventData.profile.moderator && eventData.profile.moderatorInfo) {
-                      this.moderatorsCol.add(eventData.profile);
+                      // if the moderator has an invalid currency, remove them from the list
+                      const buyerCur = !!app.serverConfig.testnet ? 'tBTC' : 'BTC';
```

Comment:
Same as above. Can we use `app.serverConfig.cryptoCurrency`?

FWIW, in my config, I'm seeing the `TBTC` as all caps. If you're seeing it sometimes come back as `tBTC`, we may need to uppercase it.